coala_quickstart/generation/SettingsClass.py

```diff
@@ -110,21 +151,27 @@ def fillup_optional_settings(self, key, funcs):
             A list of function objects i.e. either containing the run() method
             or the create_arguments() + generate_config() methods of the linter
             bears.
+        :param bear:
+            The bear object.
+        :param settings:
+            Dict of optional bear settings of the form:
+            {'setting_name': default_values,}
         """
         present_in_annot = False
-        for func in funcs:
-            inside_annot = in_annot(func, key)
-            if inside_annot is not False:
-                present_in_annot = inside_annot
-                break
+        if len(funcs) == 1:
+            present_in_annot = in_annot_recursive(bear, key)
+        else:
+            for func in funcs:
+                inside_annot = in_annot(func, key)
+                if inside_annot is not False:
```

Comment:
could have been `inside_annot`

Reply:
I didn't like that because its value is never `True`, its either `False` or some other list or something, but I think its going to do the same job